### ISSUE TITLE
Feature request: add setting for server path

### ISSUE THREAD

**Issue by alanz** [erlang-ls/vscode#8]
At the moment the `serverPath` is hardcoded as

```typescript
let serverPath = context.asAbsolutePath(path.join('erlang_ls', '_build', 'default', 'bin', 'erlang_ls'));
```

Make this a vscode setting for the plugin, so it is easy to set up an alternative (development) `erlang_ls` server.

**Comment by adam-becker** [comment]
Slightly related, but having this be more configurable (specifically how the command is invoked) is required to get the extension to work with Windows.

Running the `erlang_ls` script works on *nix, because the shell will interpret the shebang and run the script using escript. Windows doesn't like this, instead you have to run `escript` (probably whichever is available on `$PATH`) passing the `serverPath` variable as the first argument. I have verified that this does work on a Windows VM.

So we may want the entire command to be configurable. Or change the invocation to just be `escript $serverPath` since it won't break *nix operating systems.

**Comment by robertoaloi** [comment]
Solved via #22
